chat-plugins/wifi.js

```diff
@@ -413,5 +413,7 @@ var commands = {
 
 exports.commands = {
 	'giveaway': commands,
-	'ga': commands.answer
```

Comment:
Should have been `commands.guess` .
Dont parse again, just make it call the guess function.
ie. `'ga': commands.guess` works